src/issuers-verifier.test.ts

```diff
@@ -70,50 +66,19 @@ describe("isWhitelisted", () => {
 });
 
 describe("verifyAllowedIssuers", () => {
-  it("should be invalid if issuers array is empty", async () => {
```

Comment:
why's this test removed?

Reply:
so i removed this project level [util](https://github.com/Open-Attestation/verify.gov.sg/pull/68/files/d1fd3eae5e3905584c725b2a374d30a4326d6a12#diff-9504ac3cc07ed6abb1cf5f778f5e10b7cc1fe20d420bf9b2593450bd9a99dca4L45)  and replaced it with open-attestation's utils `utils.isWrappedV2Document`

with open-attestation's utils, it will first check if the document matches a OpenAttestation schema 2.0 (which it is not), failing that, it will move on to the throw code.

 since open-attestation's utils guarantees that the schema matches, we will try not to replicate the unit test at the project level.

Reply:
ok got it thanks